src/components/nodes/sequenceFlow/index.js

```diff
@@ -62,6 +65,19 @@ export default {
       }
     }
   },
+  multiplayerInspectorHandler(node, data, setNodeProp, moddle) {
+    const keys = Object.keys(data).filter((key) => key !== 'id');
+    if (keys.length <= 0) {
```

Comment:
I mean, it could be `-1` but in reality, this should be 
```suggestion
    if (keys.length === 0) {
```